collect_app/build.gradle

```diff
@@ -139,10 +139,9 @@ allprojects {
 
 dependencies {
     compile fileTree(dir: 'libs', include: '*.jar')
-    compile group: 'com.google.android.gms', name: 'play-services-analytics', version: '10.0.1'
-    compile group: 'com.google.android.gms', name: 'play-services-maps', version: '10.0.1'
-    compile group: 'com.google.android.gms', name: 'play-services-auth', version: '10.0.1'
-    compile group: 'com.android.support', name: 'multidex', version: '1.0.1'
+    compile group: 'com.google.android.gms', name: 'play-services-analytics', version: '10.2.0'
```

Comment:
I got it. No problem here. 😄  I made this PR for discussion, not for merging. I didn't refactored the commented code, because I wanted to know your opinion, before moving forward. The idea of this GSOC project is to redesign the user experience and interface. To implement the design, that I posted before, we should use Support Library. But the problem is that switching can be done only by refactoring `Activity` to `AppCompatActivity` and changing `HoloTheme` to `AppCompatTheme`. These changes are interrelated and must have for further application evolving.  So, how would you propose to approach this large issue?
And of course, thanks for the brief explanation of the situation.